recipes/catkin_pkg/recipe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "catkin_pkg" %}
+{% set version = "0.4.13" %}
+{% set sha256 = "843a8ad05e498e1eb3c9f75496d7410058af3a9d8e52b4339a8d1a6b3ff3aced" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ros-infrastructure/catkin_pkg/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  noarch: python
+  entry_points:
+    - catkin_create_pkg = catkin_pkg.cli.create_pkg:main
+    - catkin_find_pkg = catkin_pkg.cli.find_pkg:main
+    - catkin_generate_changelog = catkin_pkg.cli.generate_changelog:main_catching_runtime_error
+    - catkin_package_version = catkin_pkg.cli.package_version:main
+    - catkin_prepare_release = catkin_pkg.cli.prepare_release:main
+    - catkin_tag_changelog = catkin_pkg.cli.tag_changelog:main
+    - catkin_test_changelog = catkin_pkg.cli.test_changelog:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - docutils
```

Comment:
Are the following three really required in host?

Reply:
Alright, I reduced the dependencies as much as possible now.